fusor-ember-cli/app/controllers/review/installation.js

```diff
@@ -242,33 +242,18 @@ export default Ember.Controller.extend(NeedsDeploymentMixin, OpenshiftMixin, {
     return this.get('storageSize') + ' GB';
   }),
 
-  deploymentButtonAction: Ember.computed('hasSubscriptionsToAttach', function() {
```

Comment:
This `deploymentButtonAction` is no longer needed, as action `onDeployButton` is now used.